doc/release/1.15.0-notes.rst

```diff
@@ -2,12 +2,28 @@
 NumPy 1.15.0 Release Notes
 ==========================
 
+NumPy 1.15.0 is a release with an unusual number of cleanups, many deprecations
+of old functions, and improvements to many existing functions. Please read the
+detailed descriptions below to see if you are affected.
+
+For testing, we have switched to pytest as a replacement for the no longer
+maintained nose framework. The old nose based interface remains for downstream
+projects who may still be using it.
+
+The Python versions supported by this release are 2.7, 3.4-3.6.  The upcoming
+3.7 release should also work, but you will need to compile from source using
+Cython 0.28.2 or later. The wheels will be linked with OpenBLAS 3.0, which
+should fix some of the problems reported for NumPy 1.14.
+
 
 Highlights
 ==========
 
 * NumPy has switched to pytest for testing.
-
+* A new  `np.printoptions` context manager.
```

Comment:
in order for `np.printoptions` to become a link, it must be spelled as `numpy.printoptions`, which will then be rendered with the full module name `numpy`. There are many of these in this doc. We could do any or all of these:

- add double ` and give up on the link
- for this release convert all `np.something` to `numpy.something`
- fix the sphinx namespaces to allow `np.something`

Should I  try to do the last one for `master`?

Reply:
badly worded my question. Should I try to make a PR to modify our documentation system so that `np.printoptions` becomes a valid link, as well as `numpy.printoptions`?

Reply:
Hmm, that could be useful, I'd say go for it.